Automattic/jetpack
# Connection: Unset jetpack conection active plugins from sync callable checksum when sync is not active

Fixes https://github.com/Automattic/vulcan/issues/469

## Proposed changes:

* Ensure we unset ``$jetpack_callables_sync_checksum['jetpack_connection_active_plugins'] if the `jetpack_connection_active_plugins` option gets changed in wpcom via the no sync fallback. This way we ensure that if/when sync gets re-enabled we send the callable no matter.

### Other information:


## Jetpack product discussion


https://github.com/Automattic/vulcan/issues/469
## Does this pull request change what data or activity we track or use?



## Testing instructions:






Check steps in https://github.com/Automattic/vulcan/issues/469 and check that we get now consistent values.
